### PR TITLE
[main] port changes from 1.6 branch

### PR DESCRIPTION
**Pull request by cdeepali**
## Checklist before submitting

- [x] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/main/CONTRIBUTING.md)?
- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/main/doc/)?
- [ ] Did you write any tests to validate this change?

## Description

Fixes # (issue).

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) requests changes, they must be addressed.

### CODE REVIEW

**Review by alexlang74 — APPROVED** [review]

File: envs/openblas-env.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+      - feedstock-patches/clang/0001-Fix-clang-build-for-p10.patch    # [ppc64le]
+  - feedstock : https://github.com/conda-forge/openmp-feedstock
+    git_tag: 1d18edb1476f4c7e88eef89a75d46c9ff487a662
+    patches:
+      - feedstock-patches/openmp/0001-remove-dependency-on-openmp_mutex-and-use-GCC11-for-p10.patch
+      - feedstock-patches/openmp/0002-disable-LIBOMPTARGET-to-fix-build-on-baremetal.patch
+  - feedstock : openblas
+{% endif %}
+
+git_tag_for_env: open-ce-v1.6.0
```
Comment (npanpaliya) [inline envs/openblas-env.yaml:15]: We don't want git_tag lines in env file.

**Comment by npanpaliya** [comment]
Also please merge the latest commit in open-ce-r1.6 branch https://github.com/open-ce/open-ce/commit/f6f85acf3126e2bb27df0a4bd37f2aabc7780d36, in this PR.

File: envs/openblas-env.yaml (hunk 1 above)
Comment (cdeepali) [reply envs/openblas-env.yaml:15]: Ah yes. I will remove it.

**Comment by cdeepali** [comment]
Updated with commit - f6f85acf3126e2bb27df0a4bd37f2aabc7780d36.

**Review by npanpaliya — APPROVED** [review]

**Pull request merged**